From 8e8da772c95d70ba87cfe5f7d64792f951d78ee5 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:41:56 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(94d0c35)..lineage-16.0-android-9.0.0_r31(c7246ab) (DO NOT
 SUBMIT)

Bill Yi (1):
 - Import translations. DO NOT MERGE

Eva Bertels (1):
 - Add check for misprovisioned Pixel 2 device.

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (1):
 - Snap for 5044688 from 6d2a43ad9f6650f859520874302c4c395817d9db to
   pi-qpr2-release

Change-Id: I6ea3a28aa64e09de1b7d07ec13ef3bb94a6e8042

Change-Id: I846fa0490d222444fba0ed5e03d665ab1203e52c
---
 res/values-sw/strings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values-sw/strings.xml b/res/values-sw/strings.xml
index 87cbba2..6695f3f 100644
--- a/res/values-sw/strings.xml
+++ b/res/values-sw/strings.xml
@@ -28,5 +28,5 @@
     <string name="install_new_cert_message" msgid="4451971501142085495">"Unaweza kusakinisha vyeti kutoka katika faili PKCS#12 kwa kirefusho %1$s au %2$s kilicho katika hifadhi ya nje."</string>
     <string name="install_new_cert_button_label" msgid="903474285774077171">"Sakinisha cheti"</string>
     <string name="allow_button" msgid="3030990695030371561">"Chagua"</string>
-    <string name="deny_button" msgid="3766539809121892584">"Kana"</string>
+    <string name="deny_button" msgid="3766539809121892584">"Kataa"</string>
 </resources>
-- 
2.17.1

